src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -343,7 +375,6 @@
     <PropertyGroup Condition="'$(SkipVersionGeneration)'!='true'">
       <BuildNumberMajor >$(GeneratedBuildNumberMajor)</BuildNumberMajor>
```

Comment:
This only runs when the file doesn't exist. Though these are now redundant. I will remove them.